README.md

```
@@ -1,253 +1,307 @@
+    - **Kafka**
+      - kafka-broker
+      - kafka-rest-proxy
+      - kafka-ui
+
+4. There are two 'init' containers that will start processes to populate SQL database tables and Kafka topics. After the following containers run their process, they will automatically stop. 
+    - kafka-init
+    - sql-init
+
+5. Open a web browser and access Kafka UI. By default, the page can be accessed at `http://localhost:9095`. Click the `Topics` tab and ensure that Kafka topics exist:
+
+<img src="Documentation/Images/readme_kafkaui.png" width="75%" height="75%">
```

Markdownlint

Images should have alternate text (alt text)

(MD045, no-alt-text)